content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
@tinaafitz  I am not sure about the sequence of the events during the import but as far as I know other parts of the code we always create an entity in the db (call refresh) when the action is done.
@smelamud please respond to the question